netci.groovy

```diff
@@ -75,7 +75,7 @@ def targetGroupOsMapInnerloop = ['netcoreapp': ['Windows_NT', 'Ubuntu14.04', 'Ub
         // Archive results.
         Utilities.addArchival(newJob, '**/coverage/*,msbuild.log')
         // Timeout. Code coverage runs take longer, so we set the timeout to be longer.
-        Utilities.setJobTimeout(newJob, 180)
+        Utilities.setJobTimeout(newJob, 720)
```

Comment:
12 hrs, huh? :)  I sincerely hope it doesn't take anywhere near that long.  If it does, we should do some analysis of the tests to figure out which ones in particular are causing it to take so long.

Reply:
:) It shouldn't, it was aborted already when generating the reports so hopefully is under 6 hours, but I would like a few runs to confirm.